packages/zapp/console/src/components/LaunchPlan/SearchableLaunchPlanNameList.tsx

```diff
@@ -0,0 +1,167 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import DeviceHub from '@material-ui/icons/DeviceHub';
+import classNames from 'classnames';
+import { useNamedEntityListStyles } from 'components/common/SearchableNamedEntityList';
+import { useCommonStyles } from 'components/common/styles';
+import { separatorColor, primaryTextColor, launchPlanLabelColor } from 'components/Theme/constants';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Routes } from 'routes/routes';
+import { debounce } from 'lodash';
+import { Typography, FormGroup } from '@material-ui/core';
+import { LaunchPlanListStructureItem } from './types';
+import { SearchableInput } from '../common/SearchableList';
+import { useSearchableListState } from '../common/useSearchableListState';
+
+interface SearchableLaunchPlanNameItemProps {
+  item: LaunchPlanListStructureItem;
+}
+
+interface SearchableLaunchPlanNameListProps {
+  launchPlans: LaunchPlanListStructureItem[];
+}
+
+export const showOnHoverClass = 'showOnHover';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  container: {
+    padding: theme.spacing(2),
+    paddingRight: theme.spacing(5),
+  },
+  filterGroup: {
+    display: 'flex',
+    flexWrap: 'nowrap',
+    flexDirection: 'row',
+    margin: theme.spacing(4, 5, 2, 2),
+  },
+  itemContainer: {
+    marginBottom: 15,
+    borderRadius: 16,
+    padding: '23px 30px',
+    border: `1px solid ${separatorColor}`,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    position: 'relative',
+    // All children using the showOnHover class will be hidden until
+    // the mouse enters the container
+    [`& .${showOnHoverClass}`]: {
+      opacity: 0,
+    },
+    [`&:hover .${showOnHoverClass}`]: {
+      opacity: 1,
+    },
+  },
+  itemName: {
+    display: 'flex',
+    fontWeight: 600,
+    color: primaryTextColor,
+    marginBottom: 10,
+  },
+  itemDescriptionRow: {
+    color: '#757575',
+    marginBottom: 30,
```

Comment:
theme.spacing here and bellow for all margin values.

Reply:
fixed